ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__rosa_hcp-4.19-nightly-x86.yaml

```diff
@@ -114,6 +114,60 @@ tests:
     - chain: openshift-qe-node-density-cni
     workflow: rosa-aws-sts-hcp
   timeout: 5h0m0s
+- always_run: false
+  as: control-plane-120nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.xlarge
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      HOSTED_CP: "true"
+      INDEX_ENABLED: "true"
+      MP_LABELS: node-role.kubernetes.io/infra=
+      MP_MACHINE_TYPE: r5.4xlarge
+      MP_REPLICAS: "2"
+      MP_TAINTS: node-role.kubernetes.io/infra=:NoSchedule
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.19"
+      PPROF: "false"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "120"
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-hypershift-infra
+    - chain: openshift-qe-control-plane
+    workflow: rosa-aws-sts-hcp-mp
+  timeout: 5h0m0s
```

Comment:
`"level":"error","msg":"Process did not finish before 5h0m0s timeout"`

Please increase the timeout to 7h/8h.
(Same for control-plane-249nodes)

Reply:
yeah but it failed even before the timeout, isn't it. Executed for 20m, i will check this deprovision issue.

```
INFO[2025-05-14T03:29:01Z] Running step control-plane-120nodes-rosa-cluster-deprovision. 
{"component":"entrypoint","file":"sigs.k8s.io/prow/pkg/entrypoint/run.go:169","func":"sigs.k8s.io/prow/pkg/entrypoint.Options.ExecuteProcess","level":"error","msg":"Process did not finish before 5h0m0s timeout","severity":"error","time":"2025-05-14T03:55:26Z"}
INFO[2025-05-14T03:55:26Z] Received signal.                              signal=interrupt
INFO[2025-05-14T03:55:27Z] error: Process interrupted with signal interrupt, cancelling execution... 
INFO[2025-05-14T03:55:27Z] cleanup: Deleting release pod release-images-latest 
```